tests/pexpects/commandline.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
 from pexpect_helper import SpawnedProc, control
+from re import escape
```

Comment:
I think we should generally call this `re.escape` instead of `escape`

Reply:
Ok, will do. Thanks for all your comments, sorry it has taken me so long to reply. I'll push a new version in the New Year.